### PR TITLE
Fix: Contextual menu label missing overflow hidden

### LINKED ISSUE #26590: ContextMenuItem label missing overflow:hidden property
The style is present in v7, but for some reason removed in v8. It is required for truncation along with textOverflow and whiteSpace.

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #26590

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/587e72c3cac2e8928af887308555f062f182840d'>587e72c3cac2e8928af887308555f062f182840d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount_base.html">11509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount.html">14735</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/BaseButton-mount_base.html">1116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/BaseButton-mount.html">1142</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Breadcrumb-mount_base.html">2700</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Breadcrumb-mount.html">2953</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Checkbox-mount_base.html">2464</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Checkbox-mount.html">2658</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/CheckboxBase-mount_base.html">2236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/CheckboxBase-mount.html">2265</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ChoiceGroup-mount_base.html">4257</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ChoiceGroup-mount.html">4231</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ComboBox-mount_base.html">1194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ComboBox-mount.html">1210</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/CommandBar-mount_base.html">9260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/CommandBar-mount.html">9082</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount_base.html">11509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount.html">14735</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DefaultButton-mount_base.html">1310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DefaultButton-mount.html">1283</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRow-mount_base.html">3437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRow-mount.html">3412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRowFast-mount_base.html">3283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRowFast-mount.html">3314</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRowNoStyles-mount_base.html">3154</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DetailsRowNoStyles-mount.html">3175</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Dialog-mount_base.html">3007</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Dialog-mount.html">3118</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DocumentCardTitle-mount_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/DocumentCardTitle-mount.html">538</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Dropdown-mount_base.html">3190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Dropdown-mount.html">3080</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/FocusTrapZone-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/FocusTrapZone-mount.html">2032</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/FocusZone-mount_base.html">1858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/FocusZone-mount.html">1922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-mount_base.html">1893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-mount.html">2050</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-virtual-rerender_base.html">1088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-virtual-rerender.html">1123</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedList-virtual-rerender-with-unmount.html">1686</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-mount_base.html">557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-mount.html">570</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-virtual-rerender_base.html">556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-virtual-rerender.html">542</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">573</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/IconButton-mount_base.html">1821</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/IconButton-mount.html">1904</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Label-mount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Label-mount.html">733</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Layer-mount_base.html">4227</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Layer-mount.html">4160</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Link-mount_base.html">803</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Link-mount.html">809</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/MenuButton-mount_base.html">1643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/MenuButton-mount.html">1673</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/MessageBar-mount_base.html">2185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/MessageBar-mount.html">2221</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Nav-mount_base.html">3089</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Nav-mount.html">3047</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/OverflowSet-mount_base.html">1377</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/OverflowSet-mount.html">1369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Panel-mount_base.html">2477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Panel-mount.html">2400</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Persona-mount_base.html">1321</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Persona-mount.html">1276</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Pivot-mount_base.html">1570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Pivot-mount.html">1540</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/PrimaryButton-mount_base.html">1521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/PrimaryButton-mount.html">1549</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Rating-mount_base.html">6682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Rating-mount.html">6680</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SearchBox-mount_base.html">1487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SearchBox-mount.html">1444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Shimmer-mount_base.html">2882</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Shimmer-mount.html">2726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Slider-mount_base.html">1938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Slider-mount.html">1988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SpinButton-mount_base.html">4743</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SpinButton-mount.html">4593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Spinner-mount_base.html">799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Spinner-mount.html">811</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SplitButton-mount_base.html">3063</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SplitButton-mount.html">2988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Stack-mount_base.html">832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Stack-mount.html">808</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/StackWithIntrinsicChildren-mount_base.html">2336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/StackWithIntrinsicChildren-mount.html">2224</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/StackWithTextChildren-mount_base.html">4563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/StackWithTextChildren-mount.html">4596</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SwatchColorPicker-mount_base.html">10269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/SwatchColorPicker-mount.html">10294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TagPicker-mount_base.html">2646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TagPicker-mount.html">2425</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TeachingBubble-mount_base.html">92438</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TeachingBubble-mount.html">91146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Text-mount_base.html">779</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Text-mount.html">715</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TextField-mount_base.html">1491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/TextField-mount.html">1543</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-mount_base.html">1460</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-mount.html">1464</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-virtual-rerender_base.html">1091</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-virtual-rerender.html">985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2047</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2029</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Toggle-mount_base.html">1106</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/Toggle-mount.html">1107</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/buttonNative-mount_base.html">536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26591/perf-test/buttonNative-mount.html">548</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by spmonahan — APPROVED** [review]

**Review by smhigley — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]

**Review by pingj — APPROVED** [review]

File: change/@fluentui-react-0437e436-ce0b-49ed-b3a5-5b0478c2db87.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-0437e436-ce0b-49ed-b3a5-5b0478c2db87.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26591&commitId=5250973665569a4b15db799c71a2b05835bb5fda&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**